db/queries/base.py

```diff
@@ -16,6 +16,7 @@ def __init__(
             engine,
             transformations=None,
             name=None,
+            metadata=None
```

Comment:
If the refactor to remove `metadata` is postponed, this line has to be fixed now. `metadata` should never be `None`.

Reply:
I made the fixes, can you review it again? #1877 tracks the proposed refactoring changes.